em++.py

```diff
@@ -6,12 +6,19 @@
 
 import sys
 import emcc
+import glob
 
 emcc.run_via_emxx = True
 
 if __name__ == '__main__':
   try:
-    sys.exit(emcc.main(sys.argv))
+    argv = []
+    for i in sys.argv:
+      if '*' in i:
+        argv.extend(glob.glob(i))
```

Comment:
Maybe easier to just always use `glob.glob`? That way it will also correctly handle other glob expansions like `?` and `[...]`.

Reply:
@sbc100 Looks like you discovered this too 3 years ago on the original issue too 😅 https://github.com/emscripten-core/emscripten/issues/10197#issuecomment-573946379

Reply:
Also, please move this into a utility function in `tools/utils.py` and make it a no-op except on windows.